### PR TITLE
Update Gradle wrapper to 8.7

### PR DESCRIPTION
**Pull request by breskeby**
Update Gradle wrapper to 8.7-rc-3

### CODE REVIEW

File: x-pack/plugin/ent-search/build.gradle [hunk 1]
```diff
@@ -34,8 +34,6 @@ dependencies {
   javaRestTestImplementation(project(path: xpackModule('core')))
   javaRestTestImplementation(testArtifact(project(xpackModule('core'))))
   javaRestTestImplementation(project(':modules:lang-mustache'))
-
-  module ':modules:search-business-rules'
```
Comment (brianseeders) [inline x-pack/plugin/ent-search/build.gradle:38]: Should this be here?

Comment (brianseeders) [reply x-pack/plugin/ent-search/build.gradle:38]: :+1: just making sure it wasn't an accidental commit

**Review by brianseeders — APPROVED** [review]

**Comment by breskeby** [comment]
just a draft till 8.7 is released

**Comment by breskeby** [comment]
8.7. just got released.

**Comment by breskeby** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|7.17|[](https://github.com/elastic/elasticsearch/pull/106722)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**